packages/mistralai_dart/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: mistralai_dart
 description: Dart Client for the Mistral AI API (Mistral-7B, Mixtral 8x7B, embeddings, etc.).
-version: 0.0.2+1
+version: 0.0.2+2
```

We don't need to bump the version manually. It is done automatically when running `melos version`

```suggestion
version: 0.0.2+1
```